lib/y2_module_basetest.pm

```diff
@@ -31,8 +31,23 @@ our @EXPORT = qw(is_network_manager_default
   continue_info_network_manager_default
   accept_warning_network_manager_default
   workaround_suppress_lvm_warnings
+  with_yast_env_variables
 );
 
+=head2 with_yast_env_variables
+
+ with_yast_env_variables([extra_vars]);
+
+Set environment variables for yast application.
+C<extra_vars> extends the variables that can be used. C<extra_vars> expects a string.
+ex: with_yast_env_variables("foo=bar");
+
+=cut
+sub with_yast_env_variables {
```

Comment:
you could use a verb to name the function in general, although in this case I don't see the need for $extra_vars. It sounds to me that the real meaning would be `get_default_yast_env_vars` which would return a constant and then you add extra different parameters in the two variants of the call, and if some of them become or are the same we will refactor and add to the default set. Anyway we want to run always this in a homogeneous way. Make sense?

Reply:
Another thing, you could also use `qw(` with on element per line to create a list and put a comment to those variables and then `join` to create an string with the list. That would be helpful I think.

Reply:
I need an clarification of "run always this in a homogeneous way." I cant understand if this is a request for changes or suggestions. I would like to leave it as it is for now. The real meaning is, as you said get_default_yast_env_vars with the ability to pass additional ones in the future. For example if you need a specific env variable to run before yast2 for some other case.

Reply:
If you want the parameter maybe just rename it to `add_yast_env_variables` then inside the function you can document that some are default and the parameter add something else depending if it is graphical or console. That should be clear from the caller, right? (but it is optional, not requesting changes, it is a minor thing).

Reply:
Also despite i like the idea of the qw usage i will like to leave simple as it is now